src/updatePr.ts

```diff
@@ -21,89 +23,122 @@ declare function schedule<T>(asyncFunction: Promise<T>): void;
+                  name: scope.replace("(", "").replace(")", ""),
+                  // The color is not used and can be customized from the "label" tab in the github page
+                  color: "#FFFFFF",
+                  description: scope,
+                }),
+              () => {
+                const errorMessage = "Eror during github label update";
+                warn(errorMessage);
+                return new Error(errorMessage);
+              }
+            )
+          )
+      )
+    );
 
-      schedule(updateLabel);
+    schedule(updateLabelAction());
 
-      const ticketsSameType = pipe(
-        ticketList,
-        RA.sort(ticketOrdByType),
-        RA.uniq(ticketOrdByType)
-      );
+    const updateTitleAction = pipe(
+      configuration.updateTitle,
+      match(
+        () => TE.left(new Error("updateTitle is disabled")),
+        () => {
+          const ticketsSameType = pipe(
+            ticketList,
+            RA.sort(ticketOrdByType),
+            RA.uniq(ticketOrdByType)
+          );
 
-      if (ticketsSameType.length > 1) {
-        warn(
-          "Multiple stories with different types are associated with this Pull request.\n" +
-            "Only one tag will be added, following the order: `feature > bug > chore`"
-        );
-      }
+          if (ticketsSameType.length > 1) {
+            warn(
+              "Multiple stories with different types are associated with this Pull request.\n" +
+                "Only one tag will be added, following the order: `feature > bug > chore`"
+            );
+          }
 
-      const tag = pipe(
-        RA.head(ticketsSameType),
-        O.map((t) => t.type),
-        O.getOrElse(() => "")
-      );
+          const tag = pipe(
+            RA.head(ticketsSameType),
+            O.map((t) => t.type),
+            O.getOrElse(() => "")
+          );
 
-      const title = pipe(
-        popFromRegex,
-        ap(danger.github.pr.title),
-        ap(cleanChangelogRegex),
-        O.getOrElse(() => danger.github.pr.title)
-      );
+          const title = pipe(
+            popFromRegex,
+            ap(danger.github.pr.title),
+            ap(cleanChangelogRegex),
+            O.getOrElse(() => danger.github.pr.title)
+          );
 
-      const titleSplitter = new RegExp(/(\[.*]\s*)(.+)/g);
-      const splittingResults = titleSplitter.exec(title);
+          const titleSplitter = new RegExp(/(\[.*]\s*)(.+)/g);
+          const splittingResults = titleSplitter.exec(title);
 
-      const upperCaseTitle = pipe(
-        splittingResults,
-        O.fromNullable,
-        O.map((reg_array) =>
-          sequenceS(O.Apply)({
-            task_id: pipe(reg_array, RA.lookup(1)),
-            title: pipe(
-              reg_array,
-              RA.lookup(2),
-              O.map((s) => s.charAt(0).toUpperCase() + s.slice(1))
+          const upperCaseTitle = pipe(
+            splittingResults,
+            O.fromNullable,
+            O.map((reg_array) =>
```

Comment:
```suggestion
            O.map((regArray) =>
```

Generally, try not to leak implementation details into naming (ie. `Array`)